pages/pipelines/glossary.md

```diff
@@ -0,0 +1,72 @@
+# Pipelines glossary
+
+The following terms describe key concepts to help you use Pipelines.
+
+{:toc}
+
+## Agent
+
+An agent is a small, reliable, and cross-platform build runner that connects your infrastructure to Buildkite. It polls Buildkite for work, runs jobs, and reports results. You can install agents on local machines, cloud servers, or other remote machines. You need at least one agent to run builds.
+
+To learn more, see the [Agent overview](/docs/agent/v3).
+
+## Artifact
+
+An artifact is a file or metadata generated during a build. You can keep artifacts in a Buildkite-managed storage service or a third-party cloud storage service like Amazon S3, Google Cloud Storage, or Artifactory. Common uses include storing assets like logs and reports, or passing files between steps.
```

Comment:
"metadata" is a separate thing. While artifacts might contain meta information like timing or reports about a build or job, I think it might be good to avoid the term.

Reply:
👍🏻 I'll remove "or metadata"